src/neynar-api/common/version.ts

```diff
@@ -0,0 +1 @@
+export const version = "1.20.2";
```

Comment:
@MrKevinOConnell does this mean we have to bump up the version of this every time we ship a new version? seems extra work for every release and is prone to errors. can you read directly from package.json instead?

Reply:
it automatically updates when we run update-sdk-version.ts @manan19